nemo_skills/pipeline/openrlhf/reward_model_server.py

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Please add both nvidia and original license as well as a reference to where this code originally comes from

Reply:
Please also add some comment as to what is changed compared to original in here as well as in datasets